.github/workflows/settings-2-deploy.yml

```diff
@@ -0,0 +1,86 @@
+name: Config Settings Deployment
+on:
+  workflow_call:
+    inputs:
+      deployment-environment:
+        type: string
+        required: true
+        description: GitHub Deployment Environment Name
+      spack-type:
+        type: string
+        required: true
+        description: The type of spack deployment (eg. Prerelease, Release)
+env:
+  CONFIG_SETTINGS_PATH: ./config/settings.json
```

Comment:
The same environment variable is set in `settings-1-update.yml`. I guess the only way to share them globally is to put them in the environment, and then they're kinda hidden?

Reply:
Yeah, that is the trade off. Having to go into the settings to see the `vars` (or doing `gh variable list`), vs having it in the file...